examples/tracer-web/package.json

```diff
@@ -34,20 +34,20 @@
     "webpack-merge": "^4.2.2"
   },
   "dependencies": {
-    "@opentelemetry/api": "^0.16.0",
-    "@opentelemetry/context-zone": "^0.16.0",
-    "@opentelemetry/core": "^0.16.0",
-    "@opentelemetry/exporter-collector": "^0.16.0",
-    "@opentelemetry/exporter-zipkin": "^0.16.0",
-    "@opentelemetry/instrumentation": "^0.16.0",
-    "@opentelemetry/instrumentation-fetch": "^0.16.0",
-    "@opentelemetry/instrumentation-xml-http-request": "^0.16.0",
-    "@opentelemetry/metrics": "^0.16.0",
+    "@opentelemetry/api": "^0.17.0",
+    "@opentelemetry/context-zone": "^0.17.0",
+    "@opentelemetry/core": "^0.17.0",
+    "@opentelemetry/exporter-collector": "^0.17.0",
+    "@opentelemetry/exporter-zipkin": "^0.17.0",
+    "@opentelemetry/instrumentation": "^0.17.0",
+    "@opentelemetry/instrumentation-fetch": "^0.17.0",
+    "@opentelemetry/instrumentation-xml-http-request": "^0.17.0",
+    "@opentelemetry/metrics": "^0.17.0",
     "@opentelemetry/plugin-document-load": "^0.12.0",
     "@opentelemetry/plugin-user-interaction": "^0.12.0",
```

Comment:
0.12.0 plugins are not compatible with 0.17.0 core - but this sample was already broken as 0.16.0 is also not compatible...

Reply:
Core repo shouldn't refer to contrib plugins anyway

Reply:
after this -> https://github.com/open-telemetry/opentelemetry-js-contrib/pull/331, 2 examples can be simply removed